package.json

```diff
@@ -91,7 +91,7 @@
     "selenium-webdriver": "^4.24.1",
     "storybook": "^8.4.2",
     "vite": "^5.4.6",
-    "watch": "^1.0.2"
+    "watch": "^0.13.0"
```

Comment:
is this dependency needed since we are using esbuild/storybook that has watch inbuilt?

else move to https://www.npmjs.com/package/nsfw?activeTab=readme as its more up to day and more performant i.e. (https://www.npmjs.com/package/watch/v/0.13.0) is 10 years old :/